ui/app/AppLayouts/Profile/views/LanguageView.qml

```diff
@@ -36,14 +38,16 @@ Item {
 
     onVisibleChanged: { if(!visible) d.setViewIdleState()}
 
-    Component.onCompleted: { root.languageStore.initializeLanguageModel() }
+    Component.onCompleted: {
```

Comment:
this can also be moved to nim side? can it happen once the systems started up so UI doesn't have to handle this explicitly ?

Reply:
Yes, sure! I agree. And same than your first comment, maybe it makes more sense to have the language model in the backend where you think it is better, nim or status-go.